lib/archethic/self_repair/sync.ex

```diff
@@ -363,6 +412,31 @@ defmodule Archethic.SelfRepair.Sync do
     |> Stream.run()
   end
 
+  defp consolidate_recipients(
+         attestation = %ReplicationAttestation{
+           transaction_summary: tx_summary = %TransactionSummary{
+             version: 1,
+             movements_addresses: movements_addresses
+           }
+         },
+         %Transaction{validation_stamp: %ValidationStamp{recipients: recipients}}
+       ) do
+    consolidated_movements_addresses =
+      recipients
+      |> Task.async_stream(fn recipient ->
```

Comment:
Here also you could add `on_timeout: :kill_task` and maybe some `max_concurrency`